src/mlpack/tests/maximal_inputs_test.cpp

```diff
@@ -73,12 +71,12 @@ BOOST_AUTO_TEST_CASE(ColumnToBlocksChangeBlockSize)
   ctb.Transform(CreateMaximalInput(), output);
 
   arma::mat matlabResults;
-  matlabResults<< -3 << -3 << -3 << -3 << -3
-               << -3 << -3 << -3 << -3 << -3 << -3 << arma::endr
-               << -3 << -1 << -0.71429 << -0.42857 << -0.14286
-               << -3 << 0.14286 << 0.42857 << 0.71429 << 1 << -3 << arma::endr
-               << -3 << -3 << -3 << -3 << -3 << -3 << -3 << -3 << -3 << -3
-                  << -3 << arma::endr;
+  matlabResults = { { -3, -3,       -3,       -3,       -3, -3,      -3,
+                          -3,      -3, -3, -3 },
+                    { -3, -1, -0.71429, -0.42857, -0.14286, -3, 0.14286,
+                     0.42857, 0.71429,  1, -3 },
+                    { -3, -3,       -3,       -3,       -3, -3,      -3,
+                          -3,      -3, -3, -3 } };
```

Comment:
This looks strange, I guess in this case no need to align the values if that allows us to put the values on the same line.